template/node/src/service.rs

```diff
@@ -65,9 +94,9 @@ pub fn new_partial(config: &Configuration, manual_seal: bool) -> Result<
 		client.clone(),
 	);
 
-	if manual_seal {
+	if sealing.is_some() {
```

Comment:
A minor nit:

```suggestion
	if let Some(sealing) = sealing {
```

Then you can remove the `expect` below.

Reply:
@sorpaas yep I missed that